app/src/main/java/edu/grinnell/grinnell_publications_android/Fragments/ArticleFragment.java

```diff
@@ -0,0 +1,162 @@
+
+    /**
+     * Constructor created by the newInstance and takes in the Item ID
+     */
+    public static ArticleFragment newInstance(){
+        ArticleFragment fragment = new ArticleFragment();
+        return fragment;
+    }
+    /*
+        public static DisplayFragment newInstance(Article article){
+        DisplayFragment fragment = new DisplayFragment();
+        return fragment;
+    }
+     */
+
+    /**
+     * OnCreate for the fragment
+     */
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+
+    /**
+     * OnCreateView that inflates and sets up the view
+     */
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        final View DisplayFragmentView = inflater.inflate(R.layout.fragment_article,
+                container, false);
+
+        /* Setup view */
+        setViewItems(DisplayFragmentView);
+        toolbar.setNavigationIcon(getResources().getDrawable(R.drawable.ic_action_search)); //todo: Replace with backbutton
+        loadData();
+
+        /* logic setters */
+        setOnClickListeners();
+
+        return DisplayFragmentView;
+
+    }
+
+
+
+    /**
+     * SETUP METHOD:
+     * Takes the fragment view, sets each view item
+     */
+    private void setViewItems(View mView)
+    {
+
+        imageView = (ImageView) mView.findViewById(R.id.backdrop);
+        toolbar = (Toolbar) mView.findViewById(R.id.toolbar);
+        collapsingToolbar =
+                (CollapsingToolbarLayout) mView.findViewById(R.id.collapsing_toolbar);
+        inviteButton = (FloatingActionButton) mView.findViewById(R.id.invite);
+
+    }
+
+
+
+    /**
+     * SETUP METHOD:
+     * Sets the onclick listeners for each view item
+     */
+    public void setOnClickListeners() {
+
+        //onClick for back button
+        toolbar.setNavigationOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                getActivity().getSupportFragmentManager().popBackStack();
+            }
+        });
+
+        inviteButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                //onclick for the FAB
+            }
+        });
+
+    }
+
+    /**
+     * loads the data from article
+     */
+    private void loadData(){
+        //method for pulling data
+    }
+
+
+    /**
+     * Sets
+     */
+    private void setContent(String name, Date startDate, Date endDate, String imageURL, String description, String location){
+
+        DisplayMetrics dm = new DisplayMetrics();
+        getActivity().getWindowManager().getDefaultDisplay().getMetrics(dm);
+        int width = dm.widthPixels/4;
+        int height = dm.heightPixels/10;
+
+        collapsingToolbar.setTitle(name);
+        Picasso
+                .with(context.getContext())
```

Comment:
You could just inline this by doing `this.getContext()`. There's no need to create an object for the context if you're not reusing it.